python/src/rtree.rs

```diff
@@ -135,6 +138,12 @@ impl RTreeInner {
             Self::Float64(index) => AsRef::as_ref(index).len(),
         }
     }
+    fn buffer(&self) -> &[u8] {
+        match self {
+            Self::Float32(index) => AsRef::<[u8]>::as_ref(index),
```

Comment:
```suggestion
            Self::Float32(index) => index.as_ref(),
```
It should infer `&[u8]` output, right?